### PR TITLE
fix(docs): comparison

### LINKED ISSUE #1008: Move the docs from Github's wiki to the ./docs directory
It will be way easier for people to search something in the docs if they are part of the repo itself instead of relying on GitHub to host the docs.

### PR DESCRIPTION
**Pull request by dbritto-dev**
## Related Issues

Fixes #1008 

## Summary

Adding some improvements to comparison docs

## Check List

- [x] `yarn run prettier` for formatting code and docs

### CODE REVIEW

**Review by dai-shi — APPROVED** [review]
LGTM. Anyone else wants to review?

**Comment by dbritto-dev** [comment]
@chrisk-7777 @callumcrichton @Exalted100 what you think folks?

File: docs/getting-started/comparison.md [hunk 1]
```diff
@@ -306,8 +320,8 @@ const countAtom = atom<number>(0)
 ### Render Optimization
 
 The other difference between Zustand and Jotai is: Jotai makes render
-optimizations through atom dependency. But, with Zustand you need to do manual
-render optimizations through selectors.
+optimizations through atom dependency. While Zustand it is recommended that you
```
Comment (callumcrichton) [inline docs/getting-started/comparison.md:323]: Line 323: However, with Zustand it is recommended that you ...

**Review by callumcrichton — CHANGES_REQUESTED** [review]
Line 323: "However, with Zustand it is recommended that you ..." something along those lines would make more sense. 👍

**Comment by dbritto-dev** [comment]
@callumcrichton done

**Comment by dbritto-dev** [comment]
@chrisk-7777 @bjornstar would you mind reviewing this PR?

**Review by Exalted100 — APPROVED** [review]

**Comment by Exalted100** [comment]
@dbritto-dev just checked and it looks fine to me. Although I would personally prefer active statements like "we recommend..." over passive ones like "it is recommended..." However, it's fine either way.

File: docs/getting-started/comparison.md [hunk 2]
```diff
@@ -316,9 +330,13 @@ type State = {
   count: number
 }
 
-const useCountStore = create<State>((set) => ({
+type Actions = {
+  setCount: (countCallback: (count: State['count']) => State['count']) => void
```
Comment (bjornstar) [inline docs/getting-started/comparison.md:334]:
```suggestion
  updateCount: (countCallback: (count: State['count']) => State['count']) => void
```

**Comment by dbritto-dev** [comment]
@bjornstar @Exalted100 done

**Review by bjornstar — APPROVED** [review]
lgtm

**Pull request merged**